src/repositories_rust_services.tf

```diff
@@ -23,6 +23,36 @@ locals {
       }
     }
 
+    svc_files = {
```

Comment:
Can drop the `svc_` prefix since it's defined inside the `svc` map already.

Reply:
Fixed in https://github.com/Arrow-air/tf-github/pull/13/commits/92d3024b0ec02843802f9c6da2a1eeb6a9e3a283